lisa/tools/docker.py

```diff
@@ -65,7 +65,14 @@ def start(self, name: str = "docker") -> None:
     def _install(self) -> bool:
         service_name = "docker"
         if isinstance(self.node.os, Debian):
-            self.node.os.install_packages("docker.io")
+            try:
+                self.node.os.install_packages("docker.io")
+            except Exception as e:
+                self._log.error(
+                    f"Failed to install docker.io: {e}, trying to install docker from"
+                    " repo"
+                )
+                self._install_from_repo()
```

Comment:
How about call this method when it's Debian. and install packages for other distros? That will be more clear.

Reply:
This is not the recommended installation method. Most of the Debian distribution install using `docker.io` package.